data-table/table.js

```diff
@@ -157,6 +157,42 @@ function addColumns(numberOfColumns, index) {
      */
 }
 
+/**
+ * This function takes the id of the list container for the column names
+ * It then adds to the list another space for another column at the bottom of the list
+ * @param config id of the table
+ */
+function addColumn(config) {
```

Comment:
I'm not sure this function is necessary; consider using input.name as a cleaner way to create a label for an input field.

Reply:
I will delete this function.